apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/configuration/SecurityConfiguration.java

```diff
@@ -42,7 +41,6 @@ protected void configure(HttpSecurity http) throws Exception {
     // http.exceptionHandling().accessDeniedHandler(restAccessDeniedHandler);
   }
 
-  @Autowired
```

Comment:
Can we really remove this `@Autowired` since here we are not in the context of constructor injection?

Reply:
I added this back in